doc/source/cluster/guide.rst

```diff
@@ -42,6 +42,22 @@ To simplify Operator configuration, Ray provides a :ref:`a Helm chart <Ray-helm>
 Installing the Helm chart will create an Operator Deployment.
 The Operator manages autoscaling Ray clusters; each Ray node runs in its own K8s Pod.
 
+Handling dependencies
```

Comment:
Why not writing these contents to handling dependencies page and just have a link in this cluster page? I think that's more coherent.